src/components/molecules/TextView/TextViewRecord.tsx

```diff
@@ -1,28 +1,49 @@
 import React from 'react';
 import Box from '@material-ui/core/Box';
-import { Typography } from '../../atoms';
 import { roadIconColors, silverSmokeColor } from '../../../style';
 import SeverityImage from './SeverityImage';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography } from '../../atoms';
 
 interface IProps {
   numOfAccidents: number;
   severityDesc: string;
   imgSrc: string;
+  isLast?: boolean;
 }
 
-const red = roadIconColors.red;
+const useStyles = makeStyles(() => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'space-around',
+    width: '80%',
+    height: '20%',
+  },
+  text: {
+    position: 'relative',
+    top: '15%',
```

Comment:
?

Reply:
this is a small adjustment to put the text segment in the table align as it should be as described in zeppelin.